packages/token-bridge-sdk/src/hooks/useTransactions.ts

```diff
@@ -530,7 +530,7 @@ const useTransactions = (): [Transaction[], TransactionActions] => {
     // appends the state with a new set of transactions
     // useful when you want to display some transactions fetched from subgraph without worrying about existing state
 
-    let transactionsMap: { [id: string]: Transaction } = {}
+    const transactionsMap: { [id: string]: Transaction } = {}
```

Comment:
ts-lint error for using `let`

Reply:
When mutating `array` indexes or `object` properties, TS won't complain if you used `const`.